scripts/windows_cli_check.ps1

```
@@ -44,233 +41,170 @@ function Fail {
-  Write-Log "Vendoring Cloak/internal into go_module/modules/Cloak"
-  Copy-Item -Path $sourceDir -Destination $targetDir -Recurse
+  Remove-Item -Path $ArtifactsDir -Recurse -Force -ErrorAction SilentlyContinue
+  New-Item -ItemType Directory -Force -Path @($appDir, $serviceDir) | Out-Null
+
+  Invoke-Gh -Arguments @("run", "download", $runId, "--name", "dobbyVPN-windows.zip", "--dir", $appDir)
+  Invoke-Gh -Arguments @("run", "download", $runId, "--name", "windows_grpcvpnserver.exe", "--dir", $serviceDir)
 }
 
-function Install-Wintun {
-  $serviceDir = Join-Path $RootDir "kmp_module\services"
+function Download-Wintun {
```

PSScriptAnalyzer (1.25.0)

[warning] The cmdlet 'Download-Wintun' uses an unapproved verb.

(PSUseApprovedVerbs)